documentation/changelog.rst

```diff
@@ -18,6 +19,7 @@ New features
 Infrastructure / Support
 ----------------------
 
+* Deprecate use of flask's ``FLASK_ENV`` variable and replace it with ``FLEXMEASURES_ENV`` [see `PR #907 <https://github.com/FlexMeasures/flexmeasures/pull/907>`_]
```

Comment:
We need to move this under the 0.19 headers (I added them on main, you need to merge from there first)

Reply:
moved the changelog entry